vision/spatial_transformer/spatial_transformer.jl

```diff
@@ -0,0 +1,200 @@
+# Spatial Transformer Network
+
+#=
+In this tutorial we'll build a spatial transformer network that will transform MNIST
+digits for classification by a convolutional network
+
+* [Spatial Transformer Networks](https://proceedings.neurips.cc/paper/2015/hash/33ceb07bf4eeb3da587e268d663aba1a-Abstract.html)
+
+=#
+
+using LinearAlgebra, Statistics
+using Flux, CUDA, Zygote
```

Comment:
```suggestion
using Flux, Zygote
```
CUDA is implicitly imported.